vm/devices/storage/disk_nvme/nvme_driver/fuzz/fuzz_emulated_device.rs

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+//! A shim layer to fuzz responses from an emulated device.
+use crate::arbitrary_bool;
+use crate::get_raw_data;
+
+use arbitrary::Unstructured;
+use chipset_device::mmio::MmioIntercept;
+use chipset_device::pci::PciConfigSpace;
+use inspect::Inspect;
+use inspect::InspectMut;
+use pci_core::msi::MsiInterruptSet;
+use user_driver::DeviceBacking;
+use user_driver::emulated::{EmulatedDevice, Mapping, EmulatedDmaAllocator, DeviceSharedMemory};
+use user_driver::interrupt::DeviceInterrupt;
+
+/// An EmulatedDevice fuzzer that requires a working EmulatedDevice backend.
+pub struct FuzzEmulatedDevice<T> {
+    device: EmulatedDevice<T>,
+}
+
+impl<T: InspectMut> Inspect for FuzzEmulatedDevice<T> {
+    fn inspect(&self, req: inspect::Request<'_>) {
+        self.device.inspect(req);
+    }
+}
```

Comment:
I'm guessing we're only implementing Inspect because one of the other traits requires it right?

Can probably use the derive instead of doing it manually.

Reply:
Now using the Inspect derive